### PR TITLE
Refactor PubNubObjC creation and config access

### PR DESCRIPTION
**Pull request by wkal-pubnub**

### CODE REVIEW

File: Sources/PubNub/KMM/PubNubObjC.swift [hunk 1]
```diff
@@ -65,3 +76,25 @@ public extension PubNubObjC {
...
+
+@objc
+public class PubNubConfigurationObjC: NSObject {
+    let configuration: PubNubConfiguration
+      
+    public init(configuration: PubNubConfiguration) {
+      self.configuration = configuration
+    }
+    
+    @objc
```
Comment (jguz-pubnub) [inline Sources/PubNub/KMM/PubNubObjC.swift:90]: You can shorten the whole syntax a bit:

```
@objc
public var userId: String {
   configuration.userId
}
```

File: Sources/PubNub/KMM/PubNubObjC.swift [hunk 2]
```diff
@@ -65,3 +76,25 @@ public extension PubNubObjC {
...
+    public var userId: String {
+        @objc get { return configuration.userId }
+    }
+    
+    @objc
+    public var authKey: String? {
```
Comment (jguz-pubnub) [inline Sources/PubNub/KMM/PubNubObjC.swift:96]: The same here:

```
@objc
public var authKey: String? {
    configuration.authKey
}
```

File: Sources/PubNub/KMM/PubNubObjC.swift (hunk 1 above)
Comment (wkal-pubnub) [reply Sources/PubNub/KMM/PubNubObjC.swift:90]: done

File: Sources/PubNub/KMM/PubNubObjC.swift (hunk 2 above)
Comment (wkal-pubnub) [reply Sources/PubNub/KMM/PubNubObjC.swift:96]: done

**Review by jguz-pubnub — APPROVED** [review]

**Pull request merged**